backend/src/test/java/codezap/category/repository/CategoryRepositoryTest.java

```diff
@@ -0,0 +1,105 @@
+package codezap.category.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import codezap.category.domain.Category;
+import codezap.global.exception.CodeZapException;
+import codezap.global.repository.JpaRepositoryTest;
+import codezap.member.domain.Member;
+import codezap.member.repository.MemberRepository;
+
+@JpaRepositoryTest
+public class CategoryRepositoryTest {
```

Comment:
```suggestion
public class CategoryJpaRepositoryTest {
```
⚡ 
이전 PR 의 리뷰와 동일합니다.
우리가 테스트 하는 객체를 명확하게 명시해 주면 좋을 것 같아요

Reply:
저는 다음과 같이 생각하는데요. 

여기에 대해 짱수가 어떻게 생각하는지 궁금해요~

[SourceCodeJpaRepository 테스트 구현 PR](https://github.com/woowacourse-teams/2024-code-zap/pull/626#discussion_r1753118059)

> 저는 짱수와 조금 다른 의견인데요.
> 
> 근거는 다음과 같습니다!
> 
> 1. 클래스 애너테이션에 JPA가 명시되어있음
> 2. 실제로 테스트 대상이 되는 객체는 JPA가 아닌 인터페이스를 사용하고 있음. SourceCodeJpaRepository가 아닌 SourceCodeRepository.
> 3. JPA를 사용하지 않게 되더라도 본 테스트의 본질은 유효함. DB와 데이터를 잘 주고받을 수 있는지! 만약 그렇게 된다면 애너테이션을 제거하고 메서드를 리팩토링하면 됨.

Reply:
중복되는 논의일 것 같으니 [초롱의 PR](https://github.com/woowacourse-teams/2024-code-zap/pull/626#discussion_r1753086683) 에서 논의 이어서 진행하시죠 ㅎㅎ